src/server/scripts/EasternKingdoms/BlackrockMountain/BlackwingLair/boss_chromaggus.cpp

```diff
@@ -118,6 +121,8 @@ class boss_chromaggus : public CreatureScript
             if (id == GUID_LEVER_USER)
             {
                 _playerGUID = guid;
+                // This is here to prevent him from being pulled from the floor underneath, remove it once maps are fixed.
```

Comment:
```suggestion
                // Hack fix: This is here to prevent him from being pulled from the floor underneath, remove it once maps are fixed.
```